package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-cloud-node",
+  "version": "1.0.0",
+  "description": "Google Cloud client libraries for Node.js",
+  "main": "bin/generate-readme.js",
```

Comment:
This is going to confuse people.  I don't expect anyone to install this.  I'd much rather just have an `npm run` command